src/main/java/org/jenkinsci/plugins/oic/OicCredentials.java

```diff
@@ -48,7 +48,7 @@ public OicCredentials(
 
         if (expiresInSeconds != null && expiresInSeconds > 0) {
             long allowedClockSkewFixed = Util.fixNull(allowedClockSkewSeconds, 60L);
-            this.expiresAtMillis = (currentTimestamp + expiresInSeconds + allowedClockSkewFixed) * 1000;
+            this.expiresAtMillis = currentTimestamp + (expiresInSeconds - allowedClockSkewFixed) * 1000;
```

Comment:
shouldn't the `allowedClockSkewFixed` be added to `expiresInSeconds`?   I thought its purpose was to add some extra padding to the expiration check to allow for slightly mismatched clocks?